.github/workflows/publish.yml

```diff
@@ -0,0 +1,28 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Publish
+
+# Controls when the action will run. Triggers the workflow on push or pull request 
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master'
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Build Docker image
+        run: make build
+      - name: Push image
+        run: .scripts/github/push-image.sh reviewer-reporter
```

Comment:
this script assumes a `IMAGE_TAG` variable named like the commit, see https://github.com/libero/reviewer-submission/blob/master/.github/workflows/ci.yml#L8-L9
`make build` (or what is called inside that target) should read `IMAGE_TAG` too to tag the image appropriately

Reply:
Ah ok- will change the docker-compose file appropriately too